src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -101,6 +103,7 @@ case class MultiWorkspaceConfig(workspace_name: String,
                                 enable_unsafe_SSL: Option[Boolean]= None,
                                 thread_pool_size:  Option[Int] = None,
                                 api_waiting_time:  Option[Long] = None,
+                                mount_mapping_path: String,
```

Comment:
This is option string everywhere else, do you have a default string when instantiating ApiProxyConfig if it's not present through getOrElse or something? I didn't see that.

Reply:
![Screenshot 2023-02-20 at 10 53 58 PM](https://user-images.githubusercontent.com/69749553/220169389-40967d29-b2e4-475a-824b-2d61cf2dce2d.png)
 while building the config i am taking the mountMappingPath as  Some(config.mount_mapping_path)) and checking is as nonEmpty in later part 
![Screenshot 2023-02-20 at 10 57 58 PM](https://user-images.githubusercontent.com/69749553/220169885-9f4627cb-4f7c-4ac8-ae69-6b0d57cbf574.png)

let me know if i have to change it and make default value??i thought mount_mapping_path should not have any default value thats why i have implemented in this way

Reply:
nah this is fine. thanks for explaining.